scenes/PlayerSelection.tscn

```diff
@@ -0,0 +1,710 @@
+[gd_scene load_steps=12 format=2]
+
+[ext_resource path="res://sprites/Background_gradient.png" type="Texture" id=1]
+[ext_resource path="res://sprites/Title_logo.png" type="Texture" id=2]
+[ext_resource path="res://sprites/Player_Orange.png" type="Texture" id=3]
+[ext_resource path="res://sprites/Player_Green.png" type="Texture" id=4]
+[ext_resource path="res://sprites/Player_Purple.png" type="Texture" id=5]
+[ext_resource path="res://sprites/Player_Yellow.png" type="Texture" id=6]
+[ext_resource path="res://sprites/Player_Red.png" type="Texture" id=7]
+[ext_resource path="res://sprites/Player_Blue.png" type="Texture" id=8]
+[ext_resource path="res://sprites/AI_Mark.png" type="Texture" id=9]
+[ext_resource path="res://sprites/Human_Mark.png" type="Texture" id=10]
+
+[sub_resource type="GDScript" id=1]
+script/source = "extends Node2D
+
+var PLAYER_SETTINGS = [
```

Comment:
If you put this array into seperate file then IMO it would be better to just use this array from ARRAYS everytime, not to have it in 3 places and keep track what is the latest version. The same should be done in MainBoard.gd